src/rules/color-named/README.md

```diff
@@ -137,3 +137,45 @@ a {
   composes: red from './index.css';
 }
 ```
+
+### `ignore: ["inside-function"]`
```

Comment:
Can you move this section to *before* the `ignoreProperties` section please.

The convention is to lead with the `ignore` and `except` options.